tests/attacks/evasion/test_boundary.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import logging
+from art.attacks import BoundaryAttack
+from art import utils
+from tests.attacks import utils_attack
+from art.classifiers.classifier import Classifier
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def fix_get_mnist_subset(get_mnist_dataset):
+    (x_train_mnist, y_train_mnist), (x_test_mnist, y_test_mnist) = get_mnist_dataset
+    n_train = 10
+    n_test = 10
+    yield (x_train_mnist[:n_train], y_train_mnist[:n_train], x_test_mnist[:n_test], y_test_mnist[:n_test])
```

Comment:
Do we need to repeat this function in each test or could we reuse it from a central location?

Reply:
this is something I was thinking about too. But for the first round I decided to leave it here because for example if you look at test_fastgradient it's nearly the same except that its n_train is set to 100 instead of 10. I wasn't sure if there was a reason for this but if all or the majority of tests use the same n_train and n_test values then yes totally we should move this up to the main config file. But these are small improvements we can do very easily once the first merge is done :-)